### ISSUE TITLE
Player on the virus [Bug, question]

### ISSUE THREAD

**Issue by HAlexTM** [huytd/agar.io-clone#405]
Hi, when I go in a virus it appear to be on the virus:
![onpaste 20160501-154923](https://cloud.githubusercontent.com/assets/11839341/14942148/7627364e-0fb4-11e6-9ee0-161bc800a9ec.png)
Someone know how to fix this?
Another question: someone know how to make cells "cubic" ??

Thanks all

**Comment by dvrein** [comment]
Mine exploded to smaller particles when meet the virus. Do you make any changes to your code? Try using & build the previous revision, and see what is the difference that causing the bug.

**Comment by HAlexTM** [comment]
Yes, but I was too small to explode, anyway old builds don't help...

**Comment by HAlexTM** [comment]
..Wait a moment, I saw other branches... like the "es6" I saw some commit like virus to image, and I really like it, what's the difference between the master and the es6 branch? thank you
and for make player's cell cubic? how to change drawCircle to draw a square?

**Comment by underGhost** [comment]
@HAlexTM The difference between branch es6 and master is that master is written in es5. Also es6 has a little more refactoring.

**Comment by HAlexTM** [comment]
Thank you!

**Comment by Ayelis** [comment]
Does this issue regard the player being shown 'above' the virus instead of hidden 'below' it?

To make cells square, you would have to do collision detection differently. I believe the collision is done here by checking the distance from a cell, and not the shape of the cell. That would probably have to be changed on both the client and the server code. Also, drawing the cell would be different, which would be just in the client code.

**Comment by HAlexTM** [comment]
@Ayelis exactly

**Comment by rafaelsofizadeh** [comment]
Fuck you

5 мая 2017 г. 8:50 пользователь "Lorenzo Lapucci" <notifications@github.com>
написал:

@Ayelis <https://github.com/Ayelis> exactly

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub
<https://github.com/huytd/agar.io-clone/issues/405#issuecomment-299374904>,
or mute the thread

**Comment by HAlexTM** [comment]
@rafaelsofizadeh fuck you too but what's the problem?

**Comment by huytd** [comment]
@rafaelsofizadeh @HAlexTM please be nice on the internet.

I deleted the comments with baaddddddddd worrddddssss

**Comment by HAlexTM** [comment]
@huytd alright thanks
